build-system/compile/sources.js

```diff
@@ -141,8 +142,14 @@ const CLOSURE_SRC_GLOBS = [
   '!node_modules/core-js/modules/library/**.js',
 ].concat(COMMON_GLOBS);
 
+const THIRD_PARTY_TRANSFORM_DIRS = [
```

Comment:
For more accuracy, let’s call this `THIRD_PARTY_TRANSFORM_GLOBS`

Reply:
Renamed and left a comment also.